docs/v3.x/concepts/templates.md

```diff
@@ -0,0 +1,79 @@
+# Templates
+
+A template is a pre-made Strapi configuration designed for a specific use case. It allows you to quickly boostrap a custom Strapi app.
+
+Here are some things a template may configure for you:
+
+- Collection types and single types
+- Components and dynamic zones
+- Plugins to install, or custom plugins
+
+::: warning
+Templates and starters are not the same thing:
+
+- A _template_ is a pre-made Strapi configuration
+- A _starter_ is a pre-made frontend application that consumes a Strapi API
+  :::
+
+## Using a template
+
+You can use a template when creating a project with `create-strapi-app`.
+
+:::: tabs
+
+::: tab yarn
+
+```bash
+yarn create strapi-app my-project --template <template-github-url>
+```
+
+:::
+
+::: tab npx
+
+```bash
+npx create-strapi-app my-project --template <template-github-url>
+```
+
+:::
+
+::::
+
+You can use the `--template` option in combination with all other `create-strapi-app` options, like `--quickstart` or `--no-run`.
+
+## Creating a template
+
+To create a Strapi template, you need to publish a public GitHub repository that follows some rules.
```

Comment:
I believe a public Gitlab/Bitbucket should also work

Reply:
I could add that feature, but right now it doesn't work. I'm using a GitHub-specific url to get a zip of the repo.

I can't use `git clone` because @alexandrebodin told me we could not assume `git` was installed on the user's machine, especially for containers like Docker

Reply:
That is very true yeah, something to think about in the future if it's asked for.